phpunit.xml

```diff
@@ -15,8 +15,8 @@
         <env name="DB_CONNECTION" value="testing"/>
     </php>
     <testsuites>
-        <testsuite name="Package Test Suite">
-            <directory suffix=".php">./tests</directory>
+        <testsuite name="Package tests">
+            <directory suffix="Test.php">./tests</directory>
```

Comment:
```suggestion
            <directory>./tests</directory>
```
Redundant paramater. This is the [default value](https://github.com/sebastianbergmann/phpunit/blob/master/phpunit.xsd#L135).